Roll back apr-0.9.7 requirement to apr-0.9.7, RHEL backported fixes
seem to have addressed this requirement.

		Nico Kadel-Garcia <nkadel@gmail.com>

--- subversion-1.7.4/configure.ac.apr	2012-04-15 11:04:18.000000000 -0400
+++ subversion-1.7.4/configure.ac	2012-04-15 11:07:23.511113482 -0400
@@ -92,7 +92,7 @@
 dnl   them.  Only .7 and later can guarantee repository
 dnl   integrity with FSFS.
 
-APR_VER_REGEXES=["0\.9\.[7-9] 0\.9\.1[0-9] 1\. 2\."]
+APR_VER_REGEXES=["0\.9\.[4-9] 0\.9\.1[0-9] 1\. 2\."]
 
 SVN_LIB_APR($APR_VER_REGEXES)
 
@@ -108,7 +108,7 @@
   AC_SUBST(SVN_APRUTIL_PREFIX)
 else
   SVN_LT_SOVERSION="-version-info 0"
-  APU_VER_REGEXES=["0\.9\.[7-9] 0\.9\.1[0-9] 1\."]
+  APU_VER_REGEXES=["0\.9\.[4-9] 0\.9\.1[0-9] 1\."]
   SVN_LIB_APRUTIL($APU_VER_REGEXES)
 fi
 AC_SUBST(SVN_LT_SOVERSION)
